qiniu/auth.py

```diff
@@ -181,7 +181,10 @@ def __copy_policy(policy, to, strict_policy):
             if k in _deprecated_policy_fields:
                 raise ValueError(k + ' has deprecated')
             if (not strict_policy) or k in _policy_fields:
-                to[k] = v
+                if isinstance(v, dict):
```

Comment:
eg. now, if i build policy for upload token, i should
policy['returnBody'] = json.dumps(return_body)